style.css

```diff
@@ -1,5 +1,78 @@
 body {
 	font-family: "Red Hat Display", sans-serif;
+	background-color: #D6E1FF;
 }
 
-/* Testing */
\ No newline at end of file
+main {
+	min-height: 100vh;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.card {
+	max-width: 495px;
+	width: 100%;
+	background-color: white;
```

Comment:
Can we change this to be a hex value please.

Reply:
yes sir